Creation Login and Register Screen and install the dependencies.

CalixtoSD117 opened the pull request:
# Description

What did you implemented/Why did you implemented this:

- Design of Login Screen and Register 
- Creation of Login Screen and Register
- Intall dependencies.

## Testing

Review the Disign 


## Screenshots

<img width="434" alt="Screenshot 2024-01-19 at 6 23 36 p m" src="https://github.com/BrightCoders-Institute/s4a1-reservaci-n-de-vuelos-s4a1-reservacion-de-vuelos-team-3/assets/118943000/d1542bb6-7c16-48b0-8be8-8fc288b43f63">

CalixtoSD117 committed:
Initialize Project

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 < ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <joseocxmpo@users.noreply.github.com>

Alfredo20164920 reviewed (approved):
That's fine, but I recommend adding a border or choosing a different color for the inputs.

CalixtoSD117 committed:
Modify and finish the Login and Register Screen screens.

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 < ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
We refactored the code of some components to respect eslint rules. We also created new components to simplify the code of others.

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
We refactored the code of some views to respect eslint rules.

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
Modified gradle to implement icons on android devices

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
We add new dependencies to implement icons to the application.

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <joseocxmpo@users.noreply.github.com>

AxeLoza21 committed:
Implementation of React Navigation

We made the necessary installations to implement the navigation between screens within our application.

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
Password Text

We modified the design a little by adding a text with the specifications that the password must include

Co-authored-by: AxeLoza21 <osc.axeloza@gmail.com>
Co-authored-by: EnriqueComparan <toshio.ronin080@gmail.com>
Co-authored-by: CalixtoSD1117 <ccalixtoengelberger117@gmail.com>
Co-authored-by: José Ocampo <jocampo3@ucol.mx>

AxeLoza21 committed:
ScreenLoader moddified

AxeLoza21 committed:
ScreenLoader moddified
